### ISSUE TITLE
The Custom Device profile of the renderer .conf doesn't work for me.

### ISSUE THREAD

**Issue by valib** [UniversalMediaServer/UniversalMediaServer#585]
When I create the Custom Device profile for the renderer the UMS is not taken an attention to this and the default .conf file for the renderer is used.

**Comment by skeptical** [comment]
@valib works for me, can you give some more info?

**Comment by valib** [comment]
@skeptical when I restart the UMS the default .conf of the renderer is load not the custom one which should override the default one.

**Comment by skeptical** [comment]
@valib 
- does the custom conf exist in your profile dir, e.g. *UMS/renderers/something.conf*?
- is there anything in *debug.log*? e.g.
```logtalk
DEBUG 14:20:42.048 [cling-7] Matched media renderer "Kodi" based on dlna details "Kodi (raspberrypi) 192.168.1.6 uuid:7744ff6c-541f-48a8-0878-05fdebf240db XBMC Foundation Kodi 14.1 Git:38e4046 Kodi - Media Renderer http://kodi.tv/ http://kodi.tv/"
INFO  14:20:42.048 [cling-7] Using custom device configuration rpi.conf for uuid:7744ff6c-541f-48a8-0878-05fdebf240db
DEBUG 14:20:42.050 [cling-7] Reading RendererName: "Kodi" (default: "Unknown renderer")
DEBUG 14:20:42.050 [cling-7] Adding status button for Kodi (raspberrypi)
DEBUG 14:20:42.051 [cling-7] [rpi.conf] Reading RendererIcon: "/home/y/.config/UMS/renderers/rpi128.png" (default: "unknown.png")
```

**Comment by valib** [comment]
@skeptical yes it exists in the UMS/renderers. In the log there is nothing useful.

**Comment by skeptical** [comment]
@valib I'm sure you've checked all the obvious things, but without something to look at it's difficult to proceed :). Can you post the conf? the log?

**Comment by skeptical** [comment]
@valib fixed in b990094 (fixes my commit 82747f).  Lesson to @skeptical: don't forget to look in the mirror :-).
